python/api/cluster_serial.py

```diff
@@ -496,7 +490,7 @@ def get_requires(self, obj):
         comp_list = {}
 
         def process_requires(req_list):
-            for c in json.loads(req_list):
+            for c in req_list:
```

Comment:
According to model req_list could be None. I think we should change blank=True in model and set default value to [].